Xamarin.MacDev/Xamarin.MacDev.csproj

```
@@ -1,9 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <!--We only build for net4x on Windows due to open issue with strong namer package: https://github.com/dsplaisted/strongnamer/issues/58-->
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net472</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard2.0;net472</TargetFrameworks>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Could we go straight to net8.0?

```suggestion
    <TargetFramework>net8.0</TargetFramework>
```

---

Unfortunately we still have some projects targeting 472 referencing this, so we need to keep targeting netstandard here